js/ui/toolbar/ui.toolbar.utils.js

```diff
@@ -30,14 +35,57 @@ export function toggleItemFocusableElementTabIndex(context, item) {
             const itemInstance = getItemInstance($widget);
             const $focusTarget = widget === 'dxDropDownButton'
                 ? itemInstance._focusTarget().find(`.${BUTTON_GROUP_CLASS}`)
-                : itemInstance?._focusTarget?.() || $(itemInstance.element());
+                : itemInstance?._focusTarget?.() ?? $(itemInstance.element());
 
             const tabIndex = itemData.options?.tabIndex;
             if(isItemNotFocusable) {
                 $focusTarget.attr('tabIndex', -1);
             } else {
-                $focusTarget.attr('tabIndex', tabIndex ? tabIndex : 0);
+                $focusTarget.attr('tabIndex', tabIndex ?? 0);
             }
         }
     }
 }
+
+export function updateDimensionsInMaterial() {
```

Comment:
why did u decided to move it to utils? Seems strange to me because component state is used here

Reply:
There are no strong arguments. I will return it as component's method - https://github.com/DevExpress/DevExtreme/pull/22171/commits/660b8dc639f4e1d1e94a239c82db90d5dc9ca9ef